### PR TITLE
fix ever-growing temp folder

### PR DESCRIPTION
**Pull request by Germandrummer92**
Currently we always write to the same folder -> if you run debugtopus multiple times it will always have "more" test cases the more you use it. Instead we clean the folder before running tests

### CODE REVIEW

File: src/debugtopus.ts [hunk 1]
```diff
@@ -76,11 +76,15 @@ export const prepareTestRun = async ({
     packageRootDir = getPackageRootLevel(appDir);
   }
 
-  const tempDir = path.join(packageRootDir, "temp");
```
Comment (sinisastanic) [inline src/debugtopus.ts:79]: just curious why is temp now a string template

File: src/debugtopus.ts [hunk 2]
```diff
@@ -76,11 +76,15 @@ export const prepareTestRun = async ({
...
+  const tempDir = path.join(packageRootDir, `temp`);
 
   if (!existsSync(tempDir)) {
     await fs.mkdir(tempDir);
+  } else {
+    await fs.rm(tempDir, { force: true, recursive: true });
+    await fs.mkdir(tempDir);
```
Comment (sinisastanic) [inline src/debugtopus.ts:85]: if we are creating the tempDir in both cases, then reverse the if check and put delete there?

**Review by sinisastanic — APPROVED** [review]

File: src/debugtopus.ts (hunk 1 above)
Comment (Germandrummer92) [reply src/debugtopus.ts:79]: oopsie, wanted to add a UUID first, but complicated things, fixed

File: src/debugtopus.ts (hunk 2 above)
Comment (Germandrummer92) [reply src/debugtopus.ts:85]: true, good point! fixed

**Pull request merged**